### PR TITLE
Workshop 1 Done!

### PR DESCRIPTION
**Pull request by ZaydOSM**

### CODE REVIEW

File: __workshop/exercise-1/1-questions.md [hunk 1]
```diff
@@ -4,60 +4,74 @@
...
 ```
 
-c) [ ] `<ul></ul><img/><ol><li>one</li></ol>`
+c) [true] `<ul></ul><img/><ol><li>one</li></ol>`
 
 ## Q2 - What is a screenreader and why should we care about them?
 
-_Feel free to use the powers of Google here, but please provide link(s) to your source(s)_
+"A screen reader is a form of assistive technology (AT)[1] that renders text and image content as speech or braille output."
+https://en.wikipedia.org/wiki/Screen_reader
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:23]: Why should we care about them?

File: __workshop/exercise-1/1-questions.md [hunk 2]
```diff
@@ -4,60 +4,74 @@
...
 ## Q3 - For each of the following cases, which tags will be needed?
 
 a) You want to create a webpage with the photos from your latest vacation
+    img
 
 b) You want to create a website that lists all the art gallery websites in your city and links to their website.
+    ul, li and a
 
 c) You want to sell designer hats. You need to receive orders from the user.
+    form
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:33]: input (for user and order information), button (for submission)

File: __workshop/exercise-1/1-questions.md [hunk 3]
```diff
@@ -4,60 +4,74 @@
...
 
 ## Q8 - What is the difference between td and th?
+    td is a data entry and th is the header of the type of data:
+    Exemple in a table listing revenue per month we will have Month and Revenu in a th and the data per month in the td.
 
 ## Q9 - Which tag makes the text appear bigger: h1 or h3?
+    h1
 
 ## Q10 - In which situation can you use self closing tags?
+    area, base, br, embed, hr. iframe, img, input, link, meta, param, source, track.
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:68]: col (for tables), wbr (word break opportunity) self closing
iframe is not self closing

File: __workshop/exercise-8/index.html [hunk 4]
```diff
@@ -1 +1,39 @@
...
+            <input type="checkbox" id="extra1" name="extra" value="baby">
+            <label for="extra1">Baby seat</label>
+            <input type="checkbox" id="extra2" name="extra" value="wheelchair">
+            <label for="extra2">Wheelchair access</label>
+            <input type="checkbox" id="extra3" name="extra" value="privacy">
+            <label for="extra3">Privacy Windows</label>
+            <h2>Where would you like us to pick you up?</h2>
+            <label for="place">Pickup location :</label>
+            <select name="place" id="place">
+                <option value="public">Public space</option>
```
Comment (orangesmartie92) [inline __workshop/exercise-8/index.html:29]: missing ```<option disabled selected value>Select one</option>```

**Review by orangesmartie92 — APPROVED** [review]

File: __workshop/exercise-1/1-questions.md (hunk 1 above)
Comment (ZaydOSM) [reply __workshop/exercise-1/1-questions.md:23]: To help the visually impaired people to access to our website. We will be missing a quiet important portion of the population.

File: __workshop/exercise-1/1-questions.md (hunk 2 above)
Comment (ZaydOSM) [reply __workshop/exercise-1/1-questions.md:33]: Noted!

File: __workshop/exercise-1/1-questions.md (hunk 3 above)
Comment (ZaydOSM) [reply __workshop/exercise-1/1-questions.md:68]: Noted

File: __workshop/exercise-8/index.html (hunk 4 above)
Comment (ZaydOSM) [reply __workshop/exercise-8/index.html:29]: Thanks, I missed that one!

File: __workshop/exercise-8/index.html [hunk 5]
```diff
@@ -1 +1,39 @@
...
+            <label for="password">Password</label>
+            <input type="password" name="password" placeholder="••••••••" required />
+            <label for="email">Email</label>
+            <input type="email" name="email" placeholder="jane.doe@example.com" required />
+            <h2>Which taxi do you require?</h2>
+            <input type="radio" id="vehicle1" name="taxi" value="car" required>
+            <label for="vehicle1">Car</label>
+            <input type="radio" id="vehicle2" name="taxi" value="Van">
+            <label for="vehicle2">Van</label>
+            <input type="radio" id="vehicle3" name="taxi" value="tuk tuk">
```
Comment (orangesmartie92) [inline __workshop/exercise-8/index.html:17]: here too. missing required

File: __workshop/exercise-8/index.html [hunk 6]
```diff
@@ -1 +1,39 @@
...
+            <label for="username">Username</label>
+            <input type="text" name="username" placeholder="janedoe1" required/>
+            <label for="password">Password</label>
+            <input type="password" name="password" placeholder="••••••••" required />
+            <label for="email">Email</label>
+            <input type="email" name="email" placeholder="jane.doe@example.com" required />
+            <h2>Which taxi do you require?</h2>
+            <input type="radio" id="vehicle1" name="taxi" value="car" required>
+            <label for="vehicle1">Car</label>
+            <input type="radio" id="vehicle2" name="taxi" value="Van">
```
Comment (orangesmartie92) [inline __workshop/exercise-8/index.html:15]: also, i'm guessing you missed the required by accident

**Review by orangesmartie92 — APPROVED** [review]

File: __workshop/exercise-8/index.html (hunk 5 above)
Comment (ZaydOSM) [reply __workshop/exercise-8/index.html:17]: Not sure why but those where included in the commit 6591f8f 3 days ago.

image

**Pull request closed without merging**